bin/ch/WScriptJsrt.cpp

```diff
@@ -220,10 +217,12 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
     else
     {
         AutoString fileContent;
-        AutoString fileName;
+        char* fileName = nullptr;
+        bool freeFileName = false;
```

Comment:
rephrase this var ? I got confused as this is used for freeing the memory contained by this file.